README.md

```diff
@@ -1 +1,57 @@
-# Math-Magician
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+# Description of the project.
+- To Set up the environment and tools needed to develop a React application.
+- Configure with special linters. 
+
+# Built With
+
+- React
+- Webpack and Babel
+
+# Key Features
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+# Future Features
+
+- To add 3 pages to the website which will display different information
+
+# Getting Started
+
+**To get a local copy up and running follow these simple steps.**
+
+Run the command below from the terminal
+
+- git clone in your terminal
```

Comment:
- [ ] Kindly add the link to your [repo](git@github.com:Tekhlay/math-magicians.git) 👍

Reply:
Hi @OyePriscilla thank you for your code review. I have added the link to my repo
![teke85-Math-Magician-at-React_Setup](https://user-images.githubusercontent.com/29442846/211522520-a7219d38-3bfd-4269-8750-b4829740d3ff.png)